syntax = 'proto3';
package rstracker_pb;
import "Common.proto";

message IMUMessage {
    Vec3 gyro = 1;
    Vec3 accel = 2;
    double hardware_ts = 3;
}
